src/v/redpanda/application.cc

```diff
@@ -1073,6 +1073,10 @@ void application::start() {
 
     _admin.invoke_on_all([](admin_server& admin) { admin.set_ready(); }).get();
 
+    if (_redpanda_enabled) {
+        start_kafka(app_signal);
+    }
+
```

Comment:
This should be moved before `admin.set_ready()` just above,

Reply:
I pondered a bit about how to define readiness of a single node in a clustered service: are we ready when we're ready to talk to peers, or are we ready when we have coordinate with peers and are ready to talk to clients?

Probably the latter is more intuitive for someone reading this endpoint, so I agree -- moved these around so that we don't declare ourselves ready until we've started the kafka listener.